### ISSUE TITLE
miniZ v2.0b is out

### ISSUE THREAD

**Issue by MattF42** [RainbowMiner/RainbowMiner#2245]
https://miniz.ch/download/

**Comment by RainbowMiner** [comment]
Yes, I know. I am currently sticking with the old version, until they solve the invalid shares problem.

**Comment by RainbowMiner** [comment]
Thanks - MiniZ has been updated many times since; current master ships v2.5e3. Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
